mkt/webapps/tasks.py

```diff
@@ -383,45 +382,85 @@ def export_data(name=None):
     today = datetime.datetime.today().strftime('%Y-%m-%d')
     if name is None:
         name = today
-    root = settings.DUMPED_APPS_PATH
-    directories = ['apps']
-    for directory in directories:
-        rm_directory(os.path.join(root, directory))
-    files = directories + compile_extra_files(date=today)
+
+    # Clean up the path where we'll store the individual json files from each
+    # app dump.
+    for dirpath, dirnams, filenames in walk_storage(settings.DUMPED_APPS_PATH):
```

Comment:
It defaults to `private_storage`. I debated explicitly passing it as argument anyway, but thought this function was hard enough to read as it is.